src/core/Core.sol

```diff
@@ -31,6 +31,8 @@ contract Core is AccessControlEnumerable {
         _setRoleAdmin(Roles.LOCKER, Roles.ADMIN);
         _setRoleAdmin(Roles.MINTER_NOTARY, Roles.ADMIN);
         _setRoleAdmin(Roles.GAME_CONSUMER_NOTARY, Roles.ADMIN);
+        _setRoleAdmin(Roles.REGISTRY_OPERATOR, Roles.ADMIN);
+        _setRoleAdmin(Roles.DEPLOYER, Roles.ADMIN);
```

Comment:
Given that we rolled back to the previous deployment system, do we still need this deployer role?

Reply:
Personally I would keep this so we can build on it as in due course this will be needed